Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -118,6 +118,12 @@ public override bool OnInterceptTouchEvent(MotionEvent ev)
 
 		public override bool OnTouchEvent(MotionEvent ev)
 		{
+			if (ShouldSkipOnTouch)
+			{
+				ShouldSkipOnTouch = false;
+				return false;
```

Comment:
You can return false, you are not interested in more events. so this allow others to pick up.